Source/WebCore/dom/CustomElementRegistry.idl

```diff
@@ -31,6 +31,7 @@
 ] interface CustomElementRegistry {
     [CEReactions, Custom] undefined define(DOMString name, Function constructor);
     any get([AtomString] DOMString name);
+    [CallWith=CurrentGlobalObject] any getName(any constructor);
```

Comment:
I would be nice to add a FIXME comment explaining that the return value should be `(DOMString or undefined)` but that our bindings generator doesn't support this yet.

Reply:
Sure. I'd add a FIXME with a reference to https://bugs.webkit.org/show_bug.cgi?id=232734